### ISSUE TITLE
v0.7 - Inconsistent results

### ISSUE THREAD

**Issue by ghost** [c0xc/CapacityTester#30]
Hey mate - Overall great update with v0.7 - UI is much improved and it seems the previous issue I had is resolved.

However I did notice that I got some inconsistent results with one of my devices. It's quite possible something odd with this drive as it doesnt seem to happen to other ones but as it's consistently reproducible, thought i'd raise it.

If i format it with Windows, then it "fails" the test. If I run the test again then it will succeed.

Video below showing me reproducing it.

https://github.com/user-attachments/assets/b06cb0df-ffdc-41b5-87be-c097a6f9df45

**Comment by c0xc** [comment]
Thanks, I'm glad you find the new UI better than the old one.

What happens if you run a full test (select, then next instead of quick start)? It should tell us a bit more.

Also, there are two things you could try. After formatting, when you select it and click next, it should list that one newly formatted filesystem at the bottom. You could try to A) unmount it explicitly right before starting the test or B) remove the filesystem to really make sure nothing is trying to access it at the wrong time.
For option A, select the storage, hit next, there should be an unmount button at the bottom if it's been detected. Try that first (I'm curious, even though unmounting isn't really a concept on Windows it'll just temporarily "disconnect" it).
For option B, select the storage but don't just yet hit next, open the menu above the list on the right, go to the format tool, first option there is to clear the partition table, do that.

**Comment by ghost** [comment]
Thanks for your reply.
I was still to replicate it this morning even with a full test (see attached below) however it seems after clearing the partition table it's resolved it for this drive.
Cheers

https://github.com/user-attachments/assets/8991835c-d6b4-4c36-9fd2-d53c3855845b

**Comment by c0xc** [comment]
Thanks for the feedback, I'm glad it works for you now, after clearing the partition table.

I'm still thinking it might be some OS lock The result text I'm showing in this case is unfortunately too generic (I did add a couple of error types but when it happens at this early stage, they're not shown here). What you could do if you were really curious (other than wait for a new minor release) is start the program with a certain environment variable set which will enable logging. But it looks like this is Windows, it's probably not worth the trouble.

I'll leave this open for now. There's some other minor thing I'll have to add too, so chances are there's going to be another minor release in the near future.

**Comment by andirapandi** [comment]
(Don't want to open new ticket, probably it's related.)
I tested 3 USB sticks from different sources, all came out as fraudulent. (2x64GB, 1x128GB)
I tested then with ValiDrive and all 3 checked fine.
There probably is something seriously wrong right now.

**Comment by c0xc** [comment]
@andirapandi If it's related, then it should only happen with the format dialog of Windows open, shortly after formatting, possibly due to some lock and it should work the second time. From the OP's last comment: "it seems after clearing the partition table it's resolved it for this drive."

If it's not that, please do open a new issue and in that case, add some details like are they new and which test did you run, exactly. Include a screenshot with the error message.

**Comment by andirapandi** [comment]
Yes, the format dialog (or several, I ignored those) were open. I did not try several times.
Good to know though..
ValiDrive was working reliable, though they only recognize freshly inserted USB sticks which makes handling a little hard.
Anyhow, now I have verified my sticks and will only use it again when ordering sticks directly from China (having felt lucky..).
Thanks for answering!

**Comment by c0xc** [comment]
@andirapandi I don't have enough information to help. If you still believe that CapacityTester returned a misleading result, please provide the error message and in this particular case close the other dialog.
But since you mention that you've verified everything now, I'm assuming it's all good, closing.
